src/SeasonOrdinal.ts

```diff
@@ -0,0 +1,7 @@
+/** enum to define seasons as integers in season order */
+export default {
+  Winter: 0,
+  Spring: 1,
+  Summer: 2,
+  Fall: 3,
+} as const;
```

Comment:
It might be better to put this in a `constants/` folder or `constants.ts` file or even in `@/utilities.ts`. Not particularly opposed to this though.